p2p/transport/webtransport/listener.go

```diff
@@ -212,5 +212,14 @@ func (l *listener) Close() error {
 	l.reuseListener.Close()
 	err := l.server.Close()
 	<-l.serverClosed
+loop:
+	for {
+		select {
+		case conn := <-l.queue:
+			conn.Close()
+		default:
+			break loop
+		}
+	}
```

Comment:
why is this change required to upgrade to quic-go 0.41?

Reply:
In v0.40 quic-go changed the behaviour of Listener.Close to not close all the existing connections that were enqueued. 
See: https://github.com/quic-go/quic-go/pull/4072